.github/workflows/yarn-upgrade.yml

```diff
@@ -46,13 +46,13 @@ jobs:
         # We special-case typescript because it's not semantically versionned
         run: |-
           # Upgrade dependencies at repository root
-          ncu --upgrade --filter=@types/node,@types/fs-extra --target=minor
+          ncu --upgrade --filter=@types/node,@types/fs-extra,lambda-tester --target=minor
           ncu --upgrade --filter=typescript --target=patch
-          ncu --upgrade --reject=@types/node,@types/fs-extra,typescript
+          ncu --upgrade --reject=@types/node,@types/fs-extra,lambda-tester,typescript
           # Upgrade all the packages
-          lerna exec --parallel ncu -- --upgrade --filter=@types/node,@types/fs-extra --target=minor
+          lerna exec --parallel ncu -- --upgrade --filter=@types/node,@types/fs-extra,lambda-tester --target=minor
           lerna exec --parallel ncu -- --upgrade --filter=typescript --target=patch
-          lerna exec --parallel ncu -- --upgrade --reject='@types/node,@types/fs-extra,typescript,${{ steps.list-packages.outputs.list }}'
+          lerna exec --parallel ncu -- --upgrade --reject='@types/node,@types/fs-extra,lambda-tester,typescript,aws-sdk,${{ steps.list-packages.outputs.list }}'
```

Comment:
Why are we excluding aws sdk? Update the commit description with the reasoning.

Reply:
Added to commit message and referenced https://github.com/aws/aws-sdk-js/issues/3453